Source/WebCore/page/csp/ContentSecurityPolicy.h

```diff
@@ -291,6 +293,7 @@ class ContentSecurityPolicy final : public CanMakeThreadSafeCheckedPtr<ContentSe
     SandboxFlags m_sandboxFlags;
     bool m_overrideInlineStyleAllowed { false };
     bool m_isReportingEnabled { true };
+    bool m_isReportingToConsoleEnabled { true };
```

Comment:
Would it be worthwhile to make all the bools together and make them bit fields?

Reply:
OK